### DISCUSSION TITLE
Text not visible when Kit is opened on external monitor

### DISCUSSION THREAD

**Discussion by hakubo** [johnlindquist/kit#360] [Errors]
This is how it looks on my external monitor:
![Screenshot 2021-07-22 at 09 25 52](https://user-images.githubusercontent.com/1018759/126603269-7d0599f7-a7ab-4a05-b1f8-fa22317f9281.png)

it does work - just the text is not visible.
Unfortunatelly not 100% of times but very often.

Sometimes resizing the window helps so I'd say it is the virtualization that gets calculated wrongly somehow. But that's just a guess...

**Comment by hakubo** [comment]
Actually this sometimes happen on my main monitor as well.

**Reply by johnlindquist** [reply]
Is it only that script?
Are you launching from a keyboard shortcut?
Does running "Clear prompt cache" in the "Kit" tab fix it?

**Comment by hakubo** [comment]
renistalling Kit helped with this. 
I no longer have this issue

This issue can be closed
